starter_code/routes/movie-routes.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router  = express.Router();
+const Movie   = require("../models/movieSchema") 
+
+
+router.get('/', (req, res, next) => {
+  Movie.find()
+  .then((stuffIGetBackFromDB)=>{
+    res.render("movies-list",{movies: stuffIGetBackFromDB})
+  })
+  .catch((err) => {
+    next(err);
+    })
+  res.render('movies-list');
+});
+
+router.get("/movies/:theID", (req,res,next)=>{
+  Movie.findById(req.params.theID)
+    .then((theThingIGetBackFromID) => {
```

Comment:
this has to have the same name than what you type in res.render.
In your case would be 'theThingIGetBackFromDB' instead of 'theThingIGetBackFromID' or the other way around. The point is that they have to match